src/styles/main.scss

```diff
@@ -0,0 +1,77 @@
+$diameter: 500px;
+$center-diameter: 30px;
+
+.stop-watch {
+  border: 5px white dotted;
+  background-color: #000;
+  height: $diameter;
+  width: $diameter;
+  border-radius: 50%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    height: $center-diameter;
+    width: $center-diameter;
+    background-color: #fff;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__minute-hand,
+  &__second-hand {
+    position: absolute;
+  }
+
+  &__minute-hand {
+    background-color: blue;
+    width: 200px;
+    height: 10px;
+    top: 50%;
+    left: 50%;
+    margin-top: -5px;
+    transform-origin: left;
+    animation-duration: 3600s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    animation-name: rotateMinuteHand;
+  }
+
+  &__second-hand {
+    background-color: yellow;
+    width: 2px;
+    height: 250px;
+    top: 50%;
+    left: 50%;
+    margin-left: -1px;
+    transform-origin: top;
+    animation-duration: 60s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    animation-name: rotateSecondHand;
```

Comment:
minute-hand and second-hand has a lot of same properties. you can make mixin for it with arguments which different.